frontend/src/routes/candidate/(protected)/profile/+page.svelte

```diff
@@ -81,15 +158,15 @@
         <label for="unaffiliated" class={labelClass}>
           {$_('candidateApp.basicInfo.fields.unaffiliated')}
         </label>
-        <input id="unaffiliated" type="checkbox" class="toggle toggle-primary mr-8" checked />
+        <input id="unaffiliated" type="checkbox" class="toggle-primary toggle mr-8" checked />
       </div>
       <p class={disclaimerClass} slot="footer">
         {$_('candidateApp.basicInfo.unaffiliatedDescription')}
       </p>
     </FieldGroup>
     <FieldGroup>
-      <label for="message" class={disclaimerClass} slot="header"
-        >{$_('candidateApp.basicInfo.electionManifesto')}</label>
+      <label for="message" class={headerClass} slot="header"
+        >{$_('candidateApp.basicInfo.electionManifesto').toUpperCase()}</label>
```

Comment:
You could also just use the class `uppercase` (https://tailwindcss.com/docs/text-transform) instead of `.toUpperCase()`